store/store.go

```diff
@@ -110,6 +110,14 @@ func infoFromRemote(d snapDetails) *snap.Info {
 	}
 	info.Deltas = deltas
 
+	screenshots := make([]snap.ScreenshotInfo, 0, len(d.ScreenshotURLs))
```

Comment:
A test for this would be nice, but I think this will fall out naturally once the mock json is there.